Newsletter2Go/Export/Model/Api/Newsletter2GoItem.php

```diff
@@ -94,65 +123,71 @@ public function getItem($itemId)
+            'status' => $this->createArray('status', '', '', 'Integer'),
+            'visibility' => $this->createArray('visibility', '', '', 'Integer'),
+            'tax_class_id' => $this->createArray('tax_class_id', '', '', 'Integer'),
+            'description' => $this->createArray('description', 'Description'),
+            'short_description' => $this->createArray('short_description', 'Short Description'),
+            'meta_keyword' => $this->createArray('meta_keyword', 'Meta Keywords'),
+            'msrp' => $this->createArray('msrp', 'MSRP', 'Manufacturer\'s suggested retail price', 'Float'),
+            'news_from_date' => $this->createArray('news_from_date', '', '', 'Date'),
+            'news_to_date' => $this->createArray('news_to_date', '', '', 'Date'),
+            'custom_design_from' => $this->createArray('custom_design_from', '', '', 'Date'),
+            'custom_design_to' => $this->createArray('custom_design_to', '', '', 'Date'),
+            'is_in_stock' => $this->createArray('is_in_stock', 'Is in stock', '', 'Boolean'),
+            'qty' => $this->createArray('qty', 'Quantity', '', 'Integer'),
+            'price' => $this->createArray('price', 'Price', '', 'Float'),
+            'special_price' => $this->createArray('special_price', 'Special Price', '', 'Float'),
+            'special_from_date' => $this->createArray('special_from_date', 'Special Price From Date', '', 'Date'),
+            'special_to_date' => $this->createArray('special_to_date', 'Special Price To Date', '', 'Date'),
+            'weight' => $this->createArray('weight', 'Weight', '', 'Float'),
+            'is_salable' => $this->createArray('is_salable', 'Is Salable', '', 'Boolean'),
+            'sale' => $this->createArray('sale', 'On Sale', 'Is product on sale', 'Boolean'),
+        ];
+    }
+
+    /**
+     * @param CatalogModel\Product $product
      * @param array $productArray
-     * @param Store $store
+     * @param StoreModel\Store $store
      */
-    protected function reformatFields(Product $product, array &$productArray, Store $store)
+    protected function reformatFields(CatalogModel\Product $product, array &$productArray, StoreModel\Store $store)
     {
-        /** @var \Magento\Tax\Helper\Data $taxHelper */
-        $taxHelper = $this->om->get('Magento\Tax\Helper\Data');
+        /** @var TaxData $taxHelper */
+        $taxHelper = $this->om->get(TaxData::class);
         $taxIncluded = $taxHelper->priceIncludesTax($store);
 
-        /** @var \Magento\Tax\Model\Calculation $taxCalculation */
-        $taxCalculation = $this->om->get('Magento\Tax\Model\Calculation');
-        $request = $taxCalculation->getRateRequest(null, null, null, $store);
+        /** @var TaxCalculation $taxCalculation */
+        $taxCalculation = $this->om->get(TaxCalculation::class);
+        $rateRequest = $taxCalculation->getRateRequest(null, null, null, $store);
         $taxClassId = $product->getTaxClassId();
```

Comment:
I couldn't find the source of `getTaxClassId`, but the code seems to work anyway somehow.